frontend/src/app/components/wp-table/timeline/container/wp-timeline-container.directive.ts

```diff
@@ -66,6 +66,7 @@ import {
   groupIdFromIdentifier,
   groupTypeFromIdentifier
 } from 'core-components/wp-fast-table/builders/modes/grouped/grouped-rows-helpers';
+import {ExportTimelineService} from 'core-app/components/work-packages/exporter/ExportTimelineService';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/176e712f4acbd40158c386da3366490c4b41c3da/frontend/src/app/components/wp-table/timeline/container/wp-timeline-container.directive.ts#L70"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>